tests/model_serving/model_server/upgrade/test_upgrade_llmd_auth_kueue.py

```
@@ -0,0 +1,367 @@
+                wait_timeout=120,
+                sleep=5,
+                func=lambda: check_gated_pods_and_running_pods(
+                    labels=selector_labels,
+                    namespace=llmisvc.namespace,
+                    admin_client=admin_client,
+                ),
+            ):
+                if running_pods == 1 and gated_pods == 1:
+                    break
+        except TimeoutExpiredError:
+            assert False, (
```

Ruff (0.15.18)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)